spaces/domains.json

```
@@ -910,6 +910,7 @@
   "vote.gravityfinance.io": "gravity-finance.eth",
   "governance.xdefi.io": "xdefigovernance.eth",
   "gov.apedao.finance": "iotapes.eth",
+  "vote.bakeddao.com": "bakeddao.eth,
```

You are missing a quote after the ENS value.
```suggestion
  "vote.bakeddao.com": "bakeddao.eth",
```